apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -33,6 +39,7 @@ const Input = <T extends FieldValues>({
 	const error = errors?.[name]?.message;
 	const hasError = Boolean(error);
 	const isRadioWithOptions = type === "radio" && options?.length;
+	const shouldShowIconButton = iconName && typeof onIconClick === "function";
```

Comment:
```suggestion
	const shouldShowIconButton = iconName;
```
Icon can be non-clickable